.circleci/config.yml

```diff
@@ -37,28 +38,11 @@ _refs:
       git config credential.helper 'cache --timeout=120'
       git config user.email "$GH_EMAIL"
       git config user.name "Release Bot"
-  gus-create-change-mgmt: &gus-create-change-mgmt
-    name: 'Gus create change case management'
+  gus-prepare-environment-variables: &gus-prepare-environment-variables
+    name: 'Prepare environment variables for GUS Change Case Management'
     command: |
-      if [ "$GUS_SKIP" == "false" ]; then
-        sudo npm install -g @salesforce/change-case-management
-        export GUS_BUILD="$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"
-        echo "Gus build is ${GUS_BUILD}"
-        sfchangecase create -i ${GUS_TEMPLATE_ID} --release offcore.tooling.${GUS_BUILD} --location ${REPO_LOCATION}
-        sfchangecase check --release offcore.tooling.${GUS_BUILD} --location ${REPO_LOCATION}
-      else
-        echo "Gus change case management was skipped because GUS_SKIP is true"
-      fi
-  gus-close-change-mgmt: &gus-close-change-mgmt
-    name: 'Gus close change case management'
-    command: |
-      if [ "$GUS_SKIP" == "false" ]; then
```

Comment:
We can still skip. This is taken care of in the orb itself. See the [internal repo](https://github.com/forcedotcom/change-case-management-orb/blob/master/src/commands/check.yml#L16)